react-delegationdashboard/src/helpers/contractDataDefinitions.ts

```diff
@@ -61,13 +62,32 @@ export class Stats {
 
 export class NetworkConfig {
   topUpFactor: number;
-  topUpRewardsGradientPoint: BigInt;
-  public constructor(topUpFactor: number, topUpRewardsGradientPoint: BigInt) {
+  roundDuration: number;
+  roundsPerEpoch: number;
+  roundsPassedInCurrentEpoch: number;
+  topUpRewardsGradientPoint: BigNumber;
+  public constructor(
+    topUpFactor: number,
+    roundDuration: number,
+    roundsPerEpoch: number,
+    roundsPassedInCurrentEpoch: number,
+    topUpRewardsGradientPoint: BigNumber
+  ) {
     this.topUpFactor = topUpFactor;
+    this.roundDuration = roundDuration;
+    this.roundsPerEpoch = roundsPerEpoch;
+    this.roundsPassedInCurrentEpoch = roundsPassedInCurrentEpoch;
     this.topUpRewardsGradientPoint = topUpRewardsGradientPoint;
   }
 }
 
+export class NetworkStatus {
```

Comment:
just a question: why do we have this `NetworkStatus` that only contains `roundsPassedInCurrentEpoch` while we already have this metric in `NetworkConfig` ?

Reply:
I'll remove it

Reply:
done